docs/manual/manual_telemetry.rst

```diff
@@ -18,35 +18,39 @@ Module Support
 Speed Modules
 **************
 
-.. table:: Speed Module Support for Telemetry
+.. table:: Module Support

```

Comment:
tables with notes have to have their own special table. I still want it to match with everyone else...and it needs the 60.